db/scylla/init/init.cql

```diff
@@ -2,5 +2,8 @@ CREATE KEYSPACE sdb WITH replication = {'class': 'SimpleStrategy', 'replication_
 create TABLE sdb.car (
     id uuid PRIMARY KEY,
     full_name text,
+    car_brand text,
+    car_class text,
     movie_title text,
+    type_name text,
 );
```

Comment:
maybe use one cql file as db/cassandra/init.cql and db/cassandra/init.cql are identical

Reply:
done